PSMenu/Public/Show-Menu.ps1

```diff
@@ -55,6 +55,9 @@ is very useful.
 .PARAMETER InitialSelection
 Set initial selections if multi-select mode. This is an array of indecies.
 
+.PARAMETER Callback
+A function/scriptblock which is called every 100 milliseconds while the menu is shown
```

Comment:
Add a note that the caller should not modify the host output during this callback.

Reply:
That's not actually entirely true though, you just need to be careful if you do modify the host output.
I actually do this in one scenario, I have a status displayed above the menu that I update in the callback, I just make sure to restore the cursor position to where it was before.

Reply:
In that case, it is perhaps best to add an example call to the README.md. If you do that, then I'Il create a gif animation of it (make sure you're checked the option that the repository maintainer may commit to your branch in the pull request).

Reply:
One other though: If you want to modify the display of a menu item, then you'd use the `-MenuItemFormatter` parameter and just need to have to re-render the menu. Case in point: I'm even more curious about an example 😄

Reply:
I added a simple example to the readme.
My real life use case is basically a menu with sub-menus. The main menu displays a live status beside each menu item via `-MenuItemFormatter` and then the sub-menus display the same status above the menu using `-Callback`